docs/requirements.txt

```diff
@@ -3,7 +3,7 @@ recommonmark
 sphinx==1.7
 breathe>=4.5
 sphinxcontrib.programoutput
-sphinxcontrib.napoleon>=1.8.1
```

Comment:
@codingS3b you will need this package to build the new library `autodoc` functionality.